tests/README.md

```diff
@@ -6,3 +6,26 @@ From top-level directory run:
     $ tox
 
 **NOTE**: This requires `tox` ( please install with `pip install tox` ) and several supported versions of Python.
+
+Integration tests
+==========
+
+To run all of the integration test `modes` uncomment the following line from `tox.ini` and add the paths to your Kafka and Confluent Schema Registry instances. 
```

Comment:
I dont see any following lines?

Reply:
Do you mean addresses rather than paths?

Reply:
your words not mine :), I suppose I can update the top-level README.md as well

Reply:
No test instructions in the top level README, but simply referring to the tests/README is what I want :)